File: frontend/src/app/Business/Models/transportador.ts
import { Pessoa } from "./pessoa";
import { TipoRntrc } from "./tipoRntrc";
import { Transportador1 } from "./transportador1";


export interface Transportador  {
    id: string;
    tipoRntrcId: number;
    rntrc: string | null;
    rntrcValidade: string | null;
    dataCadastro: string | null;
    idNavigation: Pessoa;
    tipoRntrc: TipoRntrc;
    transportador1s: Transportador1[];
}

File: backend/Entities/Models/Transportador.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class Transportador
{
    public Guid Id { get; set; }

    public int TipoRntrcId { get; set; }

    public string? Rntrc { get; set; }

    public DateTime? RntrcValidade { get; set; }

    public DateTime? DataVerificacao { get; set; }

    public DateTime? DataCadastro { get; set; }

    public virtual Pessoa IdNavigation { get; set; } = null!;

    public virtual TipoRntrc TipoRntrc { get; set; } = null!;

    public virtual ICollection<Transportador1> Transportador1s { get; } = new List<Transportador1>();
}
